index.html

```diff
@@ -8,60 +8,246 @@
+              sign-ups required.
+            </p>
+          </article>
+          <ul class="skills">
+            <li>html</li>
+            <li>css</li>
+            <li>javascript</li>
+          </ul>
+          <button id="popup" type="button">See Project</button>
+        </div>
       </div>
-      <div class="heading-social">
-        <div class="icons-header">
-          <p>LET’S CONNECT</p>
+      <div class="card">
+        <img src="./assets/images/project4.png" alt="project4.png" />
+        <div class="tonic-area">
+          <h2 class="cards-details">TONIC</h2>
+          <div class="highlights">
+            <span class="highlight-text">CANOPY</span>
+            <img src="./assets/images/Counter.png" alt="Counter.png" />
+            <span class="countertext">Back End Dev</span>
+            <img src="./assets/images/Counter.png" alt="Counter.png" />
+            <span class="year">2015</span>
+          </div>
+          <article class="cards-description">
+            <p>
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required.
+            </p>
+          </article>
+          <ul class="skills">
+            <li>html</li>
+            <li>css</li>
+            <li>javascript</li>
+          </ul>
+          <button id="popup" type="button">See Project</button>
         </div>
-        <ul class="icons">
-          <li id="twitter">
-            <a href="https://twitter.com/MercyMugambi15" target="_blank"
-              ><img src="./assets/icons/twitter.png"
-            /></a>
-          </li>
-          <li>
-            <a href="https://www.linkedin.com/in/mercymugambi/" target="_blank"
-              ><img src="./assets/icons/Linkedin.png"
-            /></a>
-          </li>
-          <li>
-            <a href="https://github.com/mercymugambi" target="_blank"
-              ><img src="/assets/icons/github.png"
-            /></a>
-          </li>
-          <li>
-            <a href="#medium" target="_blank"
-              ><img src="./assets/icons/medium.png"
-            /></a>
-          </li>
-          <li>
-            <a href="#angellist" target="_blank"
-              ><img src="./assets/icons/angellist.png"
-            /></a>
-          </li>
+      </div>
+    </section>
+    <!--about-->
+    <section class="about">
+      <div class="header1">
+        <h2 class="header">
+          About<br />
+          Myself
+        </h2>
+        <article class="heading-paragraph">
+          <p>
+            Hello I’m a software developer! I can help you build a product ,
+            feature or website Look through some of my work and experience! If
+            you like what you see and have a project you need coded, don’t
+            hestiate to contact me.
+          </p>
+        </article>
+        <div class="heading-social">
+        <h2 class="icons-header">LET'S CONNECT</h2>
+        <div>
```

Comment:
As suggested by the previous reviewer
`Please use the right content and the right design here. Your About section doesn't fit the Figma design. The main differences are the spacing and the rounded corner of the About section. See below 👇🏽`

![image](https://user-images.githubusercontent.com/45739963/219826859-a553ea03-3481-4735-8d6d-f638625271fe.png)